cmake/toolschains/ubuntu-mingw64.cmake

```diff
@@ -0,0 +1,31 @@
+# Sample toolchain file for building for cross-compiling to Windows from an Ubuntu Linux system.
+#
+# Typical usage:
+#    *) install cross compiler: `sudo apt install mingw-w64`
+#    *) mkdir buildMingw64 && cd buildMingw64
+#    *) cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/Toolchain-Ubuntu-mingw64.cmake ..
+#
+
+set(CMAKE_SYSTEM_NAME Windows)
+set(TOOLCHAIN_PREFIX x86_64-w64-mingw32)
+
+# cross compilers to use for C and C++
+# Note: I had to use the posix variant to avoid compilation errors related with not finding the mutex class
+set(CMAKE_C_COMPILER ${TOOLCHAIN_PREFIX}-gcc-posix)
+set(CMAKE_CXX_COMPILER ${TOOLCHAIN_PREFIX}-g++-posix)
+set(CMAKE_RC_COMPILER ${TOOLCHAIN_PREFIX}-windres)
+
+# target environment on the build host system
+set(CMAKE_FIND_ROOT_PATH /usr/${TOOLCHAIN_PREFIX} /usr/lib/gcc/${TOOLCHAIN_PREFIX}/7.3-posix)
```

Comment:
Is there a way to not hardcode the gcc version?

Reply:
Good point. Right now I cannot think about a way to do it. Nonetheless, since this toolchain is prepared for Ubuntu, and Ubuntu only has such version of MinGW I do not think it is very important  ...

Maybe we could rename the file to `ubuntu1804-mingw64.cmake` to be more specific about the platform where the toolchain can be used.

Reply:
> Maybe we could rename the file to ubuntu1804-mingw64.cmake to be more specific about the platform where the toolchain can be used.

That's a good idea, please do that!